cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -229,39 +228,33 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             }
         }
 
-        var (ortResultInput, readDuration) = measureTimedValue { ortFile?.readValue<OrtResult>() }
-
-        ortFile?.let { file ->
-            log.perf {
-                "Read ORT result from '${file.name}' (${file.formatSizeInMib}) in ${readDuration.inMilliseconds}ms."
-            }
+        val existingOrtFile = requireNotNull(ortFile) {
```

Comment:
Moving this here could be a separate commit preceding this one.
IMO it would be nice to at least mention that: while at it this check has been moved further up to fail early.

Reply:
> Moving this here could be a separate commit preceding this one.

You're right, done.